novawallet/Modules/Staking/Validation/NominationPoolDataValidatorFactory.swift

```diff
@@ -5,6 +5,14 @@ protocol NominationPoolDataValidatorFactoryProtocol: BaseDataValidatingFactoryPr
         method: StakingSelectionMethod,
         locale: Locale
     ) -> DataValidating
+    func nominationPoolIsDestroing(
```

Comment:
I would suggest to name it in negative manner. Like, nominationPoolIsNotDestroing. Because, we allow the action when pool is not destroying.